contracts/AccountRulesList.sol

```diff
@@ -32,6 +41,22 @@ contract AccountRulesList {
         }
     }
 
+    function addAll(address[] memory accounts) internal returns (bool) {
+        bool allAdded = true;
+        for (uint i = 0; i<accounts.length; i++) {
+            if (!exists(accounts[i])) {
```

Comment:
Don't think you need the if statement here, I think you can change this to be something like 
```
bool added = add(accounts[i]);
emit AccountAdded(added, accounts[i]);
allAdded = allAdded && added;
```

Reply:
👍